doc/rst/platforms/cray.rst

```diff
@@ -204,27 +177,7 @@ Using Chapel on a Cray System
         source util/setchplenv.bash
 
 
-2) Optionally, set one or more of the following environment variables to
-   select a Chapel configuration.  These are described in greater detail
-   in :ref:`readme-chplenv`.
-
-     :``CHPL_TASKS``: tasking implementation, default ``qthreads``
-     :``CHPL_COMM``: communication implementation, default ``ugni`` on Cray
-                     XC/XE, ``gasnet`` on Cray CS
-
-   For CS\ |trade| series systems, see :ref:`readme-infiniband` for
-   information about using Chapel with InfiniBand.
-
-   The configuration selected must be one that is present in the Chapel
-   installation being used, whether that is a source distribution or the
-   pre-built module.  If it is not, the Chapel compiler will produce an
-   error message saying so when you try to compile anything.  If you get
-   this error, you will need to build the desired configuration (if you
-   are working from source) or modify your configuration so that it is
-   one of those supplied (if you are working with the pre-built module).
-
-
-3) Compile your Chapel program.  For example:
+4) Compile your Chapel program.  For example:
```

Comment:
Numbering is wonky starting here and continuing below -- this should be 2) instead of 4).

Reply:
haha, whoops. I promise I can count.. 1, 4, 5,...